spec/lib/tmuxinator/cli_spec.rb

```diff
@@ -252,8 +252,8 @@
   describe "#start" do
     before do
       ARGV.replace(["start", "foo"])
+      stub_tmux_version(1.9)
```

Comment:
I think I would prefer to leave the stub in-place in the spec, rather than externalize it to a separate file, since we're not looking at having to stub out more than the one method or doing this in a bunch of spec suites.  Perhaps using `let`s would make this more DRY, fi that's what you're going for, e.g.

```ruby
describe "#stop" do
  before do
    allow(Tmuxinator::Config).to receive_messages(version: version)
  end

  context "with tmux 1.9" do
    let(:version) { 1.9 }

    it { is_expected.to be_truthy }
  end
end
```